release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2068,6 +2068,7 @@ In various circumstances, the management state of the Cluster Samples Operator c
 
 * Before this update, the tooltip for tasks in the *PipelineRun details* showed misleading information. It showed the time elapsed since the task ran, not how long they ran. For example, it showed 122 hours for a task that ran for a couple of seconds 5 days ago. With this update, the tooltip shows the duration of the task. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011368[BZ#2011368])
 
+* Before this update, when a secret was provided for a private repository in the *Import from Git* form, the value of the secret to fetch the private repository details was not decoded. As a result, when using the *Import from Git* form to import a private Git repository, the import type and builder image were not detected. This update resolves the issue by decoding the secret value before using it. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057507[BZ#2057507])
```

Comment:
Not sure about the use of *Import from Git* form. I am not so sure it shouldn't just be *Git import* I'll have someone verify. I made a few suggestions based on minimalism and clarity.
```suggestion
* Before this update, import strategy detection did not occur when a secret was provided for a private Git repository. Consequently, the secrets value was not decoded before it is used. With his update, the secret value is now decoded before its use. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057507[BZ#2057507])
```

Reply:
done